dev-libs/libqtolm/libqtolm-3.0.1.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_P="${PN}-v${PV}"
+DESCRIPTION="A Qt wrapper for libolm"
+HOMEPAGE="https://gitlab.com/b0-matrix/libqtolm"
+
+inherit cmake
+
+SRC_URI="https://gitlab.com/b0-matrix/${PN}/-/archive/v${PV}/${PN}-v${PV}.tar.bz2 -> ${P}.tar.bz2"
+
+S=${WORKDIR}/${MY_P}
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~ppc64 ~x86"
```

Comment:
Please start them off with just ~amd64, per https://devmanual.gentoo.org/keywording/index.html#keywording-new-packages.

Reply:
Yeah, I saw that and was hesitant to do that, but I have done it now.